internal/parser/planparserv2/convert_field_data_to_generic_value.go

```diff
@@ -0,0 +1,74 @@
+package planparserv2
```

Comment:
now we first convert `*schemapb.TemplateValue` to `*planpb.GenericValue` in this file, then convert again in fill_expression_value.go from `*planpb.GenericValue` to `*planpb.GenericValue`(with some expression specific checks). 

is it possible to do it just once? maybe in fill_expression_value.go: directly convert `*schemapb.TemplateValue` to `*planpb.GenericValue` in various types of exprs? then we should be able to get rid of this file completely.

I could be wrong

Reply:
Yes, as you mentioned, it's only necessary to convert to the required type again when special checks are needed. The overhead for this is negligible. The current approach might make the code look a bit clearer, don't you think?

Reply:
Because we still can't directly use the field type to retrieve elements from `schemapb.TemplateValue`. For example, `FloatField > {min}`, with `{"min": 5}`. When passing it, `5` is definitely represented as an `int64` type. We still need to first determine the type of templateValue, then retrieve the element, and attempt to convert it to the field's type.